_posts/2022-01-18-system-ref-webinar-available.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Watch and try demo materials from HPP and OpenAMP System Reference Webinar
+author: openamp
+layout: post
+date: 2023-01-18 12:00:00
+description: >-
+  The Heterogenous Processing Project (HPP) and OpenAMP System Reference working group webinar and supporting materials are posted to the Linaro resources page.
+category: news
+tags:
+  - OpenAMP
+  - Webinar
+  - HPP
+  - Open Source
+  - System-DT
+image: /assets/images/content/openAMP_share_image.png
+---
+The Heterogenous Processing Project (HPP) and OpenAMP System Reference working group webinar was on December 13, 2022.  The video and supporting materials are posted to the Linaro resources page.
+
+The webinar covered progress that has been made over the last 3 years with a special focus on current work.  
+
+Three areas were highlighted:
+* Upstream build and use with the Linux kernel and Zephyr
+* Virtio devices other than rpmsg on CPU cores that do not share a hypervisor
+* System Devicetree hands-on with the Looper tool
+
+Each area has slides, a demo, and links to resources that attendees can use to reproduce the demos on their own and try things out after the event.  
+
+Speakers and Panelists:
+* Bill Mills, Principal Technical Consultant, Linaro
+* Arnaud Pouliquen, Design S/W Senior Engineer, STMicroelectronics
+* Tanmay Shah, Sr. Software Development Engineer, AMD Xilinx
+* Dan Milea, Technology Office, Wind River
+* Tomas Evensen, Senior Fellow Open Source Program Office, AMD Xilinx
+* Bruce Ashfield, Lopper / Yocto Maintainer, AMD Xilinx
+* Mathieu Poirier, Kernel Maintainer, Linaro
+* Tammy Leino, Software Engineer, Siemens
+
+
+The Linaro resource page with the video of the presentation and the slides can be found [here](https://resources.linaro.org/en/resource/2QSzYLK9Uxq4CFhVBYAnvb).
+In the posted slides are links to the documentation and repositories for reproducing the demos, as well as a link to join the OpenAMP community Discord channel.
```

Comment:
What about adding the link to the the Dicsord channel in the "Useful Links" on  https://www.openampproject.org/  page?

Reply:
@arnopo , per Bill's suggestion I'll make it a different PR to update the Useful Links.  I will file issues for these items so I don't forget.
The last commit was required because the GitHub Actions run a check for broken links.  CHAOSS keeps moving their code of conduct around.  I updated to the new link in PR #95 and merged that without review.  The last commit pulls that change in so that GitHub Actions for this PR pass.  https://github.com/OpenAMP/website/commit/8825c7169736720d84ab3ef6861a479003e946f6 is the diff.